src/components/answers/index.js

```diff
@@ -322,7 +322,7 @@ function Answers({ question, answers, loading, source }) {
                     <div className="answers--item__content--right">
                       {!_.isNil(answerWithFeedback.confidenceScore) ? (
                         <div className="answers--item__confidence">
-                          Confidence:
+                          Score:
```

Comment:
@seirasto I think we should remove the % since it no longer confidence.

Reply:
I actually prefer it like this. Can we leave it?

Reply:
It will confusing since we call it score and multiplying it with 100 and adding % sign implies `confidence`?

Reply:
Since the score is normalized it will either be 0-100% or 0-1 I prefer 0-100. What do others think? @bhavani105 @avisil

Reply:
@seirasto I woud prefer without per @kpfadnis without % and to 2 decimal places

Reply:
@seirasto @bhavani105 @avisil Merging with unresolved conversation as required for release.